index.d.ts

```diff
@@ -0,0 +1,31 @@
+declare module 'resin-jwt' {
+	import Strategy from 'passport-jwt';
+
+	interface StrategyOpts {
+		secret: string;
+		apiKeys: {[key: string]: string};
+		apiHost: string;
+		apiPort: string;
+	}
+
+	interface CreateServiceJwtOpts {
+		service: string;
+		apikey: string;
+		secret: string;
+		payload?: any;
+		expiry?: number;
+	}
+
+	interface RequestUserJwtOpts {
+		apiHost: string;
+		apiPort: string;
+		token: string;
+		userId?: string;
+		username?: string;
+	}
+
+	export function strategy(opts: StrategyOpts): Strategy;
+	export function createJwt(payload: any, secret: string, expiry?: number): string;
+	export function createServiceJwt(opts: CreateServiceJwtOpts): string;
```

Comment:
Unrelated to this PR, it's funny that `createServiceJwt` accepts a full blown object for mostly the same arguments `createJwt` accepts as positional arguments -- love how consistent we are :)

Reply:
Yeah I noticed that as well... And the similarity of the options objects, just with different names :p. I'm not sure what this is being changed for, but if it's going to be used more maybe we should invest some time into cleaning up the interface @wrboyce? That would be a breaking change though ofc.

Reply:
I guess we could just create a whole new interface, fair enough. Agreed it's above the scope of this PR though.